plugins/AuctionMaster/config.yml

```
@@ -0,0 +1,335 @@
+#Example: a player with permission 'auctionmaster.limit.bids.6' can have maximum 6
+#bids at once
+use-bid-limit: false
+bid-limit-reached-message: '&cHas alcanzado tu límite máximo de ofertaa. No puedea ofertar en otra subasta hasta que obtenga una.'
+
+#If you set this to true, the plugin will begin to check
+#players for the permission 'auctionmaster.limit.auctions.number'
+#where number is the maximum number of auctions a player can have at the same time.
+#If the permission is not found, it will be set to the maximum (28)
+#If a player has 2 permissions, the bigger number has priority
+#Example: a player with permission 'auctionmaster.limit.auctions.6' can have maximum 6
+#auctions at once
+use-auction-limit: false
+auction-limit-reached-message: '&cHas alcanzado tu límite máximo de subastas. No puedes crear otra subasta hasta que recolectes una..'
+
+#If you set this to true, the plugin will begin to check
+#players for the permission 'auctionmaster.limit.duration.number'
+#where number is the maximum number of hours a player can have on an auction.
+#If a player has 2 permissions, the bigger number has priority
+#Example: a player with permission 'auctionmaster.limit.duration.6' can have a maximum 6
+#hours long auction
+#ATTENTION! Don't set your default duration over a limit you are giving because
+#the plugin WILL NOT check for it.
+#ATTENTION AGAIN! You can limit the number of hours up to 48 hours, the plugin will not check for more
+use-duration-limit: false
+duration-limit-reached-message: '&cLa duración seleccionada supera su límite. Ajústelo automáticamente a su máximo.'
+
+
+#Message added to the category item's lore when you are browsing it
+category-browsing: '&aCurrently browsing!'
+
+#Message added to the category item's lore when you are not browsing it
+category-no-browsing: '&eHaz click para ver los items!'
+
+#ATTENTION: Please don't use the same name for items in the menu
+#or the same name for the menus themselves, some checks are made
+#based on name so you can bug the system by giving the same name to
+#two items in the same inventory
+
+go-back-item: '262'
+go-back-item-name: '&aIr atras'
+go-back-item-lore:
+  - '&7Ir atras al menu de pujas.'
+
+previous-page-item: '262'
+#Use %page-number% for the current page's number
+previous-page-item-name: '&aPage %page-number%'
+previous-page-item-lore:
+  - '&7Click para ir a'
+  - '&7la pagina de antes.'
+
+next-page-item: '262'
+#Use %page-number% for the current page's number
+next-page-item-name: '&aPage %page-number%'
+next-page-item-lore:
+  - '&7Click para ir a'
+  - '&7la siguiente pagina.'
+
+starting-menu-name: '&8Menu de pujas'
+close-menu-item: '166'
+close-menu-item-name: '&cCerrar Menu'
+close-menu-item-lore:
+  - '&7Cerrar Menu'
+
+browsing-menu-name: '&8Navegador de subastas'
+browsing-menu-item: '41'
+browsing-menu-item-name: '&6Navegador de subastas'
+browsing-menu-item-lore:
+  - '&7Busca artículos a la venta por'
+  - '&7jugadores alrrededor del server!'
```

```suggestion
  - '&7jugadores alrededor del server'
```
No veo necesario los signos de exclamación